apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -264,12 +265,12 @@ class SectionTable extends Component {
         }
       },
       {
-        property: 'editDelete',
+        property: 'buttonCell',
```

Comment:
nit: Naming on this could be something more generic like `actionCell` since I've seen some designs that put these in a drop down instead of buttons.

Reply:
Although I could be thinking of designs for a different page.

Reply:
You are right. I'll probably try to address in a future PR as I have a string of them coming in this space.